crates/contracts/Cargo.toml

```diff
@@ -16,24 +16,24 @@ required-features = ["bin"]
 default = []
 bin = [
     "anyhow",
-    "env_logger",
     "ethcontract-generate",
-    "log",
     "serde_json",
+    "tracing",
+    "tracing-subscriber",
 ]
 
 [dependencies]
-ethcontract = { version = "0.21.0", default-features = false }
-serde = "1.0"
+ethcontract = { workspace = true }
+serde = { workspace = true }
 
 # [bin-dependencies]
-anyhow = { version = "1.0", optional = true }
-env_logger = { version = "0.9", optional = true }
-ethcontract-generate = { version = "0.21.0", optional = true, default-features = false, features = ["http"] }
-log = { version = "0.4", optional = true }
-serde_json = { version = "1.0", optional = true }
+anyhow = { workspace = true, optional = true }
+ethcontract-generate = { workspace = true, optional = true, features = ["http"] }
+serde_json = { workspace = true, optional = true }
+tracing = { workspace = true, optional = true }
+tracing-subscriber = { workspace = true, features = ["env-filter", "fmt"], optional = true }
```

Comment:
A non-trivial change, this was the only use of `log` crate, so I just changed it to use `tracing` instead for consistency with our other crates.